### PR TITLE
Closes #141: Updated labels and added disclaimer 

### LINKED ISSUE #141: Documentation:  Add COEQ disclaimer text to `qs_metabolic` documentation
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

The disclaimer text got lost when the `qs_metabolic` data was moved from asdmiralmetabolic to pharmaversesdtm. This should be added back in `inst/sdtms-spec` so that it renders in the documentation

### PR DESCRIPTION
**Pull request by Siddhesh2097**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: R/qs_metabolic.R [hunk 1]
```diff
@@ -3,12 +3,12 @@
 # Manual edits are not recommended, as changes may be overwritten.
 #' Questionnaire Dataset-metabolic
 #'
-#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies
+#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following ([link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq)).
```
Comment (manciniedoardo) [inline R/qs_metabolic.R:6]:
```suggestion
#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).
```

File: R/qs_metabolic.R [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
 #'
 #' @name qs_metabolic
 #' @title Questionnaire Dataset-metabolic
 #' @keywords dataset
-#' @description A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies
+#' @description A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following ([link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq)).
```
Comment (manciniedoardo) [inline R/qs_metabolic.R:11]:
```suggestion
#' @description A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).
```

File: inst/extdata/sdtms-specs.json [hunk 3]
```diff
@@ -323,7 +323,7 @@
     {
       "name": "qs_metabolic",
       "label": "Questionnaire Dataset-metabolic",
-      "description": "A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies",
+      "description": "A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following ([link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq)).",
```
Comment (manciniedoardo) [inline inst/extdata/sdtms-specs.json:326]:
```suggestion
      "description": "A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).",
```

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Looks good, just suggested removing a pair of brackets.

File: R/qs_metabolic.R [hunk 4]
```diff
@@ -3,12 +3,12 @@
 # Manual edits are not recommended, as changes may be overwritten.
 #' Questionnaire Dataset-metabolic
 #'
-#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies
+#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).
```
Comment (AndersAskeland) [inline R/qs_metabolic.R:6]:
```suggestion
#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {pharmaversesdtm} as well as the ADCOEQ code are for not-for-profit use only within pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).
```

I cant quite remember, but I think this needs to be changed in [inst/extdata/sdtms-specs.json](https://github.com/pharmaverse/pharmaversesdtm/pull/144/files#diff-5db1936bda11ada4ca8d0a7083f7e554fd054c553858068d1884c60c11ba9f90)?

**Review by AndersAskeland — CHANGES_REQUESTED** [review]
I guess we would need to change this to pharmaversesdtm? @manciniedoardo Was the actual text pre-approved or can we change it?

File: R/qs_metabolic.R (hunk 4 above)
Comment (manciniedoardo) [reply R/qs_metabolic.R:6]: @AndersAskeland ah yes good spot. I confirm that it needs to be changed not in `qs_metabolic.R` but `inst/extdata/sdtms-specs.json`. I've left a suggestion below @Siddhesh2097 but you'll have to regenerate the documentation. Text was approved but again none of the substance is changing, just the location of the data within the wider pharmaverse. I think it's fine, but equally @AndersAskeland if you'd like to give a courtesy email to UoLeeds letting them know of the move, maybe that'll cover us on all bases.

Comment (Siddhesh2097) [reply R/qs_metabolic.R]: @AndersAskeland thank you for noticing this. It's updated now
@manciniedoardo

Comment (manciniedoardo) [reply R/qs_metabolic.R]: @AndersAskeland let us know how you want to proceed!

**Review by AndersAskeland — APPROVED** [review]
I think its LGTM now. In relation to the email, It'll probably just confuse UoLeeds if they get a main about such a small change. Feel free to merge @manciniedoardo
